game_sprites.py

```diff
@@ -424,11 +425,12 @@ def shoot(self):
             fade_speed=self.shot_fade_speed,
             wrap_max_x=self.screen_width,
             wrap_max_y=self.screen_height,
+            speed_scale=self.speed_scale,
             sound=BonusUFO.sound_fire)
 
         self.shot_list.append(new_ufo_shot)
 
-        self.shoot_timer = self.fire_rate + self.fire_rate_mod
+        self.shoot_timer = (self.fire_rate + self.fire_rate_mod) / self.speed_scale
```

Comment:
Hmmmm... I don't think this is correct. The timer should not use _speed_scale_. The speed of decrease/increase will, however, be affected.